recipes/libell/libell_git.oe

```diff
@@ -0,0 +1,5 @@
+require libell.inc
+
+COMMIT = "branch=master"
```

Comment:
please use a commit so we don't end up with random build failures with this recipe